spec/requests/api/v1/topics_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'Topics', type: :request do
+  describe 'displays list of topics' do
+    let(:parsed_response) { JSON.parse(response.body) }
+
+    subject { get api_v1_topics_path, as: :json }
+
+    context 'when there are topics' do
+      let(:topics_number) { Faker::Number.between(from: 1, to: 100) }
+
+      before do
+        create_list(:topic, topics_number)
+        subject
+      end
+
+      it 'returns all persisted topics' do
+        expect(parsed_response.size).to eq(topics_number)
+      end
```

Comment:
we should add a case in where we check the data returned by the controller (that it includes  `id`, `name` and `image_url`)